pkg/server/router/manifest.go

```diff
@@ -181,19 +180,20 @@ func (mr ManifestRouter) DeleteManifest(ctx context.Context, w http.ResponseWrit
 type SubmitApplicationRequest struct {
 	Application manifestsdk.CredentialApplication `json:"application" validate:"required"`
 	// Once we have JWT signed wrapper that can get the did this can be removed
-	RequesterDID string `json:"requesterDid" validate:"required"`
+	ApplicantDID string `json:"applicantDid" validate:"required"`
 }
 
 func (sar SubmitApplicationRequest) ToServiceRequest() manifest.SubmitApplicationRequest {
 	return manifest.SubmitApplicationRequest{
 		Application:  sar.Application,
-		RequesterDID: sar.RequesterDID,
+		ApplicantDID: sar.ApplicantDID,
 	}
 }
 
 type SubmitApplicationResponse struct {
-	Response    manifestsdk.CredentialResponse    `json:"response"`
-	Credentials []credential.VerifiableCredential `json:"credentials"`
+	Response manifestsdk.CredentialResponse `json:"credential_response"`
```

Comment:
Other responses are camelCase right?

Reply:
this is tough because it is required by the spec: https://identity.foundation/credential-manifest/#embed-locations-2